README.md

```diff
@@ -256,6 +264,50 @@ Adding explicit `it` calls works around this error:
  }
 ```
 
+### Common workflow: locking other source sets or configurations
+
+By default, we lock the `main` source set of java projects as production dependencies, and any source set that ends in 
+`test` as test dependencies. This behaviour can be configured per project via the `versionsLock` extension.
+
+For instance, the default behaviour (assuming you only have the default `main` and `test` source sets) can be replicated through:
+
+```gradle
+versionsLock {
+    disableJavaPluginDefaults()  // do not lock any configuration / source set by default
+    production { from sourceSets.main }
+    test { from sourceSets.test }
+}
```

Comment:
I think all snippets should be copy-and-pasteable with some useful purpose - since this just replaces thedefaults, don't think it's worth including